ui/pages/confirmations/components/confirm/info/personal-sign/personal-sign.tsx

```diff
@@ -33,9 +33,25 @@ const PersonalSignInfo: React.FC = () => {
   }
 
   const { from } = currentConfirmation.msgParams;
+  const isSiweSigReq = isSIWESignatureRequest(currentConfirmation);
 
   return (
     <>
+      {isSiweSigReq && (
```

Comment:
As with the permit simulation, could this warrant its own component such as `SIWESimulation`?

Not necessarily a separate file, could also just be local to this file.